romancal/regtest/test_mos_skycell_pipeline.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import roman_datamodels as rdm
+
+from romancal.pipeline.mosaic_pipeline import MosaicPipeline
+
+from .regtestdata import compare_asdf
+
+# mark all tests in this module
+pytestmark = [pytest.mark.bigdata, pytest.mark.soctests]
+
+
+@pytest.fixture(scope="module")
+def run_mos(rtdata_module):
+    rtdata = rtdata_module
+
+    # Test Pipeline
+    rtdata.get_asn("WFI/image/L3_mosaic_asn.json")
+    output = "r0099101001001001001_r274dp63x31y81_prompt_F158_i2d.asdf"
+    rtdata.output = output
+    args = [
+        "roman_mos",
+        rtdata.input,
+    ]
+    MosaicPipeline.from_cmdline(args)
+    rtdata.get_truth(f"truth/WFI/image/{output}")
+    return rtdata
+
+
+@pytest.fixture(scope="module")
+def output_filename(run_mos):
+    return run_mos.output
+
+
+@pytest.fixture(scope="module")
+def output_model(output_filename):
+    with rdm.open(output_filename) as model:
+        yield model
+
+
+@pytest.fixture(scope="module")
+def truth_filename(run_mos):
+    return run_mos.truth
+
+
+def test_output_matches_truth(output_filename, truth_filename, ignore_asdf_paths):
+    diff = compare_asdf(output_filename, truth_filename, **ignore_asdf_paths)
+    assert diff.identical, diff.report()
+
+
+def test_resample_ran(output_model):
+    # DMS373 test output is resampled to a skycell
+    # FIXME this doesn't test if the output is a skyceell
+    assert output_model.meta.cal_step.resample == "COMPLETE"
```

Comment:
The previous code:
https://github.com/spacetelescope/romancal/blob/e741d1963e163d8fbbeff714cb6a2acea662ae8a/romancal/regtest/test_mos_pipeline.py#L151-L154
mentions that the requirement is that the output is resampled to a skycell. The test doesn't appear to test that (and was not previously an assert).

Reply:
I agree that the comment is misleading.  Let's leave this for now.  Perhaps the test should be updated to look for https://github.com/spacetelescope/rad/blob/main/src/rad/resources/schemas/mosaic_basic-1.0.0.yaml#L133 , but I'm not sure that we actually populate that right now.  I'll spin that into a separate issue.